doc/setting-up.md

```diff
@@ -0,0 +1,159 @@
+Setting Up
+========
+part of [Qcode Documentation](index.md)
+
+* * *
+
+This document will guide through pulling together features from the qcode-tcl library to set up request handling for Naviserver that will include validation of user input and authenticating users. 
+
+### Naviserver Initialization
+Firstly, we'll need to register filters and procs as well as creating an anonymous session but only when the server starts up. There are a few filters provided by the library and here we will use three of them:
+
```

Comment:
Firstly, we'll need to register filters and procs and ensure that an anonymous session has been created. 

There are a few filters provided by the library and here we will use three of them: